### ISSUE TITLE
Allow overloading Dolphin Actions

### ISSUE THREAD

**Issue by netopyr** [canoo/dolphin-platform#144]
Overloading methods is a standard feature of modern OO. Dolphin Actions should allow it, too.

**Comment by hendrikebbers** [comment]
You want to overload the action of a superclass in the controllers on the server, right?

**Comment by netopyr** [comment]
No, what you describe sounds more like overriding. I want the same action name with different parameters.

**Comment by hendrikebbers** [comment]
lol, maybe it's too early in the morning :)
